apps/test/unit/templates/teacherDashboard/EditSectionFormTest.js

```diff
@@ -47,29 +47,45 @@ describe('EditSectionForm', () => {
         .props().value,
       SectionLoginType.picture
     );
-    assert.equal(
-      loginTypeField
-        .find('option')
-        .at(2)
-        .props().value,
-      SectionLoginType.email
+  });
+  it('renders LoginTypeField with word and picture options for word sections - NO students', () => {
+    const wrapper = mount(
+      <EditSectionForm
+        title="Edit section details"
+        handleSave={() => {}}
+        handleClose={() => {}}
+        editSectionProperties={() => {}}
+        validGrades={['K', '1', '2', '3']}
+        validAssignments={validAssignments}
+        assignmentFamilies={assignmentFamilies}
+        sections={{}}
+        section={noStudentsSection}
+        isSaveInProgress={false}
+        stageExtrasAvailable={() => false}
+        hiddenStageState={{}}
+        updateHiddenScript={() => {}}
+        assignedScriptName="script name"
+      />
     );
+    const loginTypeField = wrapper.find('LoginTypeField');
+    assert.equal(loginTypeField.length, 1);
+    assert.equal(loginTypeField.find('option').length, 2);
     assert.equal(
       loginTypeField
         .find('option')
-        .at(3)
+        .at(0)
         .props().value,
-      SectionLoginType.google_classroom
+      SectionLoginType.word
     );
     assert.equal(
       loginTypeField
         .find('option')
-        .at(4)
+        .at(1)
         .props().value,
-      SectionLoginType.clever
+      SectionLoginType.picture
     );
   });
-  it('renders LoginTypeField with word and picture options for word sections', () => {
+  it('renders LoginTypeField with word and picture options for picture sections - students', () => {
```

Comment:
I think this would be more clear to say "with students" instead of just the dash - you can infer the meaning if you look at the other test names with no students, but on its own is not obvious.